docs/stories/components/Button.stories.js

```diff
@@ -44,7 +44,7 @@ import { Button } from '@buffetjs/core';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import {
   faPlus,
-  faTrash,
+  faTrashAlt,
```

Comment:
You don't need this import in the example as you don't use it in the code. Same for the `faBaseBall`

Reply:
We use all of them in the code below

Reply:
If I am not mistaken these imports concern the Example which only uses the `faTrash` the imports for the story are set in the top of the file.

Reply:
If you want to leave them you can add more examples though.

Reply:
Ok just understood